### PR TITLE
feat: hugo init 

### PR DESCRIPTION
**Pull request by VishnuSanal**
this PR contains the implementation of the current design using hugo & initialization of CI for auto deployments.

someone with admins access: please select source => github actions under build & deployment section from settings => pages (https://github.com/KochiFOSS/kochifoss.github.io/settings/pages)

### CODE REVIEW

**Comment by VishnuSanal** [comment]
live deployment: https://vishnusanal.github.io/kochifoss.github.io/

**Review by subins2000 — APPROVED** [review]

File: .hugo_build.lock
Comment (subins2000) [inline .hugo_build.lock:1]: This file should be in `.gitignore`

**Review by subins2000 — CHANGES_REQUESTED** [review]
@VishnuSanal _a One change needed, LGTM

**Comment by subins2000** [comment]
Awesome, the site on Hugo is now up :rocket: https://kochifoss.org/ Thanks @VishnuSanal !

**Pull request merged**